### ISSUE TITLE
Training on cityscapes, acc always under 0.1

### ISSUE THREAD

**Issue by lucasjinreal** [AlexMa011/pytorch-polygon-rnn#7]
Using cityscapes for training, the accuracy calculated always under 0.1, I think this implementation is broken. Any suggestions?

**Comment by AlexMa011** [comment]
Maybe you should carefully adjust the lr and batch size. I use lr=0.0001 and batch size 8. You can set a larger lr at first.

**Comment by lucasjinreal** [comment]
how many epochs you have trained and final accuracy about how many?

**Comment by AlexMa011** [comment]
about 80K  and the accuracy of the training set can be up to 95%.

**Comment by AlexMa011** [comment]
If you have no further problem, I will close this issue.
